### ISSUE TITLE
UI confusion: Forwarding History blank white due to unexpected keyboard focus

### ISSUE THREAD

**Issue by wtogami** [Ride-The-Lightning/RTL#1256]
At the bottom of the **Routing** -> **Forwarding History** page is the **Events per page** selector.

![image](https://github.com/Ride-The-Lightning/RTL/assets/93665/61cf7f9f-da73-47f6-b11c-8f5b38f3562d)

It defaults to 10. Often when you click on it to select a different value the page goes blank white which is confusing. If you attempt to scroll the page with up and down arrows it often nothing happens. It is confusing because page scrolling unexpectedly doesn't have keyboard focus. Keyboard up and down arrows is controlling the **Events per page** selector.
